files/spark.yaml

```diff
@@ -0,0 +1,13 @@
+log:
```

Comment:
Is this just an artefact from the traefik rock? Looks suspiciously like the traefik config file 😅

Reply:
Yes, just created a placeholder :sweat_smile: . I still need to figure out what goes in here or do we even need this.

Reply:
ack. removed.